addons/test_mail/tests/test_performance.py

```diff
@@ -452,6 +458,8 @@ def test_message_subscribe_subtypes(self):
     @warmup
     def test_message_track(self):
         record = self.env['mail.performance.tracking'].create({'name': 'Zizizatestname'})
+        record.flush()
```

Comment:
Not using the new method?

Reply:
Actually not necessary. I will just keep the initial flush_tracking done in setup, other flushes are done by warmup / query counters. I will update commit message accordingly.